### ISSUE TITLE
WCAG2ICT Project: Add Contributors and editors to the document

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#21]
- [x] Add former editors
- [x] Add Phil Day and Chris Loiselle to editors list
- [x] Move contributors from 2013 document to a _Previous contributors_ section 
- [x] Add in the list of contributors from our current TF into the list of contributors (when known)

**Comment by ChrisLoiselle** [comment]
@shawnthompson or @michael-n-cooper 

1.  do we have a list of former editors that I can pull from? 
2.  is this something that can be automatically pulled in ?
3. for the contributors and current contributors, can we pull that data directly from the WCAG2ICT group member list or should I be referencing elsewhere?

**Comment by GreggVan** [comment]
Should be on the old report - no? 

gregg

------------------------------
Gregg Vanderheiden
***@***.***

**Comment by ChrisLoiselle** [comment]
@GreggVan Thank you, great point. I've made a pull request to add a markdown file to the main branch. Confirming with @maryjom per editor process now, thank you !

**Comment by shawnthompson** [comment]
@ChrisLoiselle, wrong Shawn @mentioned?

**Comment by ChrisLoiselle** [comment]
Perhaps, apologies.

**Comment by maryjom** [comment]
@ChrisLoiselle In the previous [WCAG2ICT note](https://www.w3.org/TR/wcag2ict/), there is a list of editors and a list of authors. From what I can tell, editor/author lists are managed in the file respec-config.js. The [W3C style guide](https://www.w3.org/Guide/manual-of-style/#Editors) gives tips on including former editors, but doesn't say if this is optional or required.  I'm sure @michael-n-cooper or @daniel-montalvo could answer that.

If former editors are required/recommended, the code to use for adding former editors would be added after the editors (see example usage in WCAG :
`formerEditors: [
		{
			name: "FirstName LastName",
			company: "Company name",
			w3cid: 5DigitIDCode
		},
                {
			name: "Mary Jo Mueller",
			company: "IBM",
			w3cid: 33602
		}
	],`

Here's the authors markup to put in that same file...after former editors:
`authors:  [
      { name: "FirstName LastName", url: "http://example.org/",
        company: "Company name", companyURI: "http://example.com/" },
      { name: "FirstName2 LastName2", url: "http://example2.org/",
        company: "Company name2", companyURI: "http://example2.com/" }
],`

**Comment by maryjom** [comment]
WCAG also has an [Acknowledgements appendix](https://www.w3.org/TR/WCAG21/#acknowledgments) with some sub-sections that recognize "Participants of the AG WG active in the development of this document", "Other previously active WCAG WG participants and other contributors to WCAG 2.0, WCAG 2.1, or supporting resources", and "Enabling Funders". Here's a link to the [WCAG source files](https://github.com/w3c/wcag/tree/main/acknowledgements) for those sections so you can see the verbiage they use and methods for listing the participants.

The old WCAG2ICT had similar [Acknowledgement sections](https://www.w3.org/TR/wcag2ict/#acknowledgements). So if we wind up listing previous contributors you have the full list.

I'll have a conversation with Daniel and Chuck on Monday, 6 February to see what is required and/or recommended for us to do in the updated Working Group Note.

**Comment by ChrisLoiselle** [comment]
@maryjom thanks, just noting that I've read what you have stated and I'll monitor.

**Comment by ChrisLoiselle** [comment]
updates made via https://github.com/w3c/wcag2ict/pull/100 , @maryjom  let me know if this is correct.

**Comment by maryjom** [comment]
@daniel-montalvo I have merged in Chris' pull request to add in former editors as this looks good. Please comment on this issue what we should have in the updated Note's [Acknowledgement sections](https://www.w3.org/TR/wcag2ict/#acknowledgements). I've added checkboxes below where you can indicate what we should have.

Acknowledgements for current WCAG2ICT work and contributions:
- [ ] authors and/or contributors (and should those be at the top of the document under the editors as it was in the previous Note?)
- [ ] participants in the Task Force
- [ ] active participants in the AG WG who reviewed/approved the Note
- [ ] enabling funders

Do we need to continue to include any of the following acknowledgements from previous WCAG2ICT published Note:
- [ ] previous participants in the Task Force
- [ ] previous authors
- [ ] previous active participants in the AG WG who reviewed/approved the Note 
- [ ] previous enabling funders (if the list is different from what we have for the current work)

For current participants and authors/contributors, can we obtain the list of using GitHub commands or commands to obtain TF participant list, or will we have to manually set up?

**Comment by ChrisLoiselle** [comment]
@daniel-montalvo and @maryjom let me know what you'd like me to take action on regarding this issue, thank you!

**Comment by daniel-montalvo** [comment]
Hi @maryjom  @ChrisLoiselle  sorry for my late response.

My proposal would be.

* [x] Editors, provided by respecConfig variable and done by @ChrisLoiselle 
* [x] Former editors (at the top of the document, provided by respecConfig variable and done by @ChrisLoiselle 
* [ ] Acknowledgement section (manually from a MD file)
  * [ ] Previous contributors
  * [ ] Current participants in the WCAG2ICT Task Force
  * [ ] Current participants in the WCAG Working Group that have actively reviewed and contributed
  * [ ] Enabling founders
 I would suggest to not list the previous TF and WG participants for the 2013 version, i thikn we could do a better job of synthesis by just listing the previous contributors. We'd have already quite a long section there anyway.

If you all agree with this, I could create the structure and the necessary md files and then @ChrisLoiselle and/or @maryjom  could fil them in.

**Comment by maryjom** [comment]
Oops....didn't mean to close this since the work is partially complete.

**Comment by maryjom** [comment]
@daniel-montalvo Are there enabling funders that need to be named? If so, what are they?

**Comment by daniel-montalvo** [comment]
Thanks @maryjom 

That is a good catch. We should have that section as well. I'll will work on it.

**Comment by ChrisLoiselle** [comment]
@daniel-montalvo  or @maryjom  are there any other open items on this? Original thread checkboxes appear to be done but there are commented checkboxes that are now still outstanding. Please let me know if there any action items to be assigned to me within the sub threaded comments.

**Comment by maryjom** [comment]
@ChrisLoiselle I'm going to address updating the lists again right before publishing the FPWD with a new issue - see #130. For now you can see that the [AG WG participants section](https://w3c.github.io/wcag2ict/#participants-in-the-ag-working-group-that-actively-reviewed-and-contributed) is in there with the folks that have been posting in our issues as we work on them. Will update with AG WG participants that responded to surveys before we publish the final note too. See issue #131.

I opened a separate issue for Daniel to add the enabling funders section, but he hasn't gotten to that yet. See #121. Other than that, the initial info in there is sufficient. Daniel seemed to think we didn't need the full list of AG WG members.

**Comment by maryjom** [comment]
All up-to-date and published in the FPWD. Will open a new issue for the next draft to be published with WCAG 2.2 incorporated.
